package.json

```diff
@@ -114,7 +113,17 @@
     "pouchdb-adapter-node-websql": "^6.4.1",
     "pouchdb-adapter-leveldb": "^6.4.1"
   },
-  "browser": {
```

Comment:
Moved this to `pouchdb-req-http-query/package.json` which is where I should have put it in a first place!